lib/resources/mssql_session.rb

```diff
@@ -24,31 +30,59 @@ class MssqlSession < Inspec.resource(1)
 
     def initialize(opts = {})
       @user = opts[:user]
-      @pass = opts[:pass]
+      @password = opts[:password] || opts[:pass]
+      if opts[:pass]
+        warn '[DEPRECATED] use `password` option to supply password instead of `pass`'
+      end
       @host = opts[:host] || 'localhost'
       @instance = opts[:instance]
+
+      # check if sqlcmd is available
+      skip_resource('sqlcmd is missing') if !inspec.command('sqlcmd').exist?
     end
 
     def query(q)
       escaped_query = q.gsub(/\\/, '\\\\').gsub(/"/, '\\"').gsub(/\$/, '\\$').gsub(/\@/, '`@')
-      cmd_string = "sqlcmd -Q \"#{escaped_query}\""
-      cmd_string += " -U #{@user} -P #{@pass}" unless @user.nil? or @pass.nil?
+      # surpress 'x rows affected' in SQLCMD with 'set nocount on;'
+      cmd_string = "sqlcmd -Q \"set nocount on; #{escaped_query}\" -W -w 1024 -s ','"
+      cmd_string += " -U #{@user} -P '#{@password}'" unless @user.nil? or @password.nil?
       if @instance.nil?
         cmd_string += " -S #{@host}"
       else
         cmd_string += " -S #{@host}\\#{@instance}"
       end
-      puts cmd_string
       cmd = inspec.command(cmd_string)
       out = cmd.stdout + "\n" + cmd.stderr
       if out =~ /Sqlcmd: Error/
-        skip_resource("Can't connect to the MS SQL Server.")
+        return skip_resource("Can't connect to the MS SQL Server.")
       end
-      cmd
+      SQLQueryResult.new(cmd, parse_result(cmd))
```

Comment:
`parse_result` doesn't exist... did you mean `parse_csv_result` which is below in line 68?